articles/azure/azs-how-config-vm-update-management.md

```diff
@@ -0,0 +1,204 @@
+
+    - Choose a **Name**, **Subscription**, **Resource Group**, **Location** and ensure *Create Azure Run As Account* is set to **Yes**.
+
+    - Click **Create**.
+
+        ![Add Automation Account](images/azs-browser-add-automation-account.png)
+
+        > [!NOTE]
+        > The error "Azure Classic Run As account creation error" may occur. This **DOES NOT** affect the process.
+
+4. Once deployed, navigate to the *Automation Account*. In the new blade, Under *Configuration Management*, select the solution to enable (Inventory & Change Tracking or Update Management).
+
+    - Select the *Log Analytics workspace* to link the automation account to.
+
+    - Click **Enable**.
+
+    - Wait for the deployment to complete.
+
+    - Repeat step 4 for both Inventory & Change Tracking and Update Management.
+
+        > [!TIP]
+        > Leave the public Azure portal open; you'll need to use it again later.
+
+5. Log in to the [Azure Stack portal](https://portal.frn00006.azure.ukcloud.com).
+
+6. Navigate to the VM that you want to enable **Update & Management** on and under *Settings*, select the *Extensions* blade.
+
+    > [!WARNING]
+    > For any monitoring to work correctly, the VM **must** have HTTPS (Port 443) enabled in the **Network Security Group** rules.
+
+7. Click **Add** at the top and select the extension **Azure Monitor, Update and Configuration Management**, then click **Create**.
+
+    ![VM enable Update and Management](images/azs-browser-log-analytics-enable-update-management.png)
+
+    - Provide the extension with the **Workspace ID** and **Primary Key** values (noted down previously).
+
+    - Repeat step 7 for the **Azure Monitor Dependency Agent**.
+
+8. Once the deployment is finished, head back to the public Azure portal, navigate to the *Log Analytics workspace* -> *Advanced settings* -> *`VM Type` Servers*.
+
+    - You will see `x` number of `VM Type` COMPUTERS CONNECTED, depending on how many VMs you linked to the workspace.
+
+        ![Log Analytics workspace computers connected](images/azs-browser-log-analytics-workspace-computers-connected.png)
+
+    - Click *x VM Type COMPUTERS CONNECTED*. This will bring up logs for the *Log Analytics workspace*.
+
+    - Change the time range to **Last 30 minutes**.
+
+9. Execute the following log query, passing in your `VM TYPE`:
+
+    - `Heartbeat | where OSType == "VMTYPE" | summarize arg_max(TimeGenerated, *) by SourceComputerId | top 500000 by Computer asc | render table`
+
+    - Example for Linux VM: 
+
+        `Heartbeat | where OSType == "Linux" | summarize arg_max(TimeGenerated, *) by SourceComputerId | top 500000 by Computer asc | render table`
+
+    - If your VM shows in the results, you have successfully linked your VM from Azure Stack to your *Log Analytics workspace*.
+
+        - [See here for more information](https://docs.microsoft.com/en-us/azure/automation/automation-update-management#confirm-that-non-azure-machines-are-onboarded)
```

Comment:
Can this link be added to the end of the line above? i.e. ...from Azure Stack to your *Log Analytics workspace* (See here for more information).